scipy/sparse/_compressed.py

```diff
@@ -91,6 +93,8 @@ def __init__(self, arg1, shape=None, dtype=None, copy=False):
             coo = self._coo_container(arg1, dtype=dtype)
             arrays = coo._coo_to_compressed(self._swap)
             self.indptr, self.indices, self.data, self._shape = arrays
+            if not self.has_canonical_format:
+                self.sum_duplicates()
```

Comment:
This shouldn't be needed, because we don't produce duplicate indices when initializing a COO container from a dense input.

Reply:
Good point! :)   Thanks!!